astropy/io/fits/tests/test_header.py

```diff
@@ -1845,7 +1844,7 @@ def test_invalid_float_cards(self):
         assert (str(h.cards['FOCALLEN']) ==
                 _pad("FOCALLEN= +1.550000000000E+002"))
         assert (str(h.cards['APERTURE']) ==
-                     _pad("APERTURE= +0.000000000000E+000"))
+                _pad("APERTURE= +0.000000000000E+000"))
```

Comment:
these all fit in one line again, and is more readible that way

Reply:
Just fixing indentation, if you want that we reformat all the code to 100 chars there are other tools.

Reply:
no it's fine here, but I really prefer to leave the lines alone that only got broken up in this PR. Certainly the breaking the lines at 80 char can be disabled in the tools you used.